services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -47,6 +52,7 @@ const createCheckboxList = ({
 export const ContractDetailsSummarySection = ({
     submission,
     navigateTo,
+    disableDownload,
```

Comment:
Rather than surface a new prop why not use the submission data  itself to determine whether or not to display the button?

I worry about dependencies between the React props and the submission data in our UI in general. Whenever possible its good to rely on the submission (or the URL) to figure out what UI to display. I suggest only add a React prop for truly _new_ data that cannot be calculated otherswise (such as the "navigateTo` link href here).

In this case I think we could make cont similar to `isSubmitted` and `isEditing` for `isPreviousSubmission`.

Reply:
^^ if we can't figure that out, I suggest naming the `disableDownload` button prop to `isPreviousSubmission` since I believe that is the context it is really capturing that is new.